fetch.bs

```diff
@@ -2911,6 +2972,39 @@ with a <i>CORS flag</i> and <i>recursive flag</i>, run these steps:
    <li><a lt="should response to request be blocked due to nosniff">should <var>internalResponse</var> to <var>request</var> be blocked due to nosniff</a>
   </ul>
 
+ <li>
+  <p>If <var>internalResponse</var>'s <a for=response>status</a> is <code>206</code>,
+  <var>response</var>'s <a for=response>type</a> is "<code>opaque</code>",
+  <var>internalResponse</var>'s <a for=response>range-requested flag</a> is set, and
+  <var>request</var>'s <a for=request>header list</a> does not <a for="header list">contain</a>
+  `<code>Range</code>`, then:
+
+  <ol>
+   <li><p>Set <var>internalResponse</var> to a new <a for=/>response</a>.
+
+   <li><p>Set <var>response</var>'s <a>internal response</a> to <var>internalResponse</var>.
```

Comment:
Didn't that already happen in the above step? This resets all state though. Does that really work with the various algorithms?

Reply:
In the previous step, I've changed what *internalResponse* refers to, but *response's* *internal response* will still refer to the previous response.

Or would it be better to set *response* to a new opaque response, then set *internalResponse* to its *internal response*?

Reply:
I think reusing the CORB strategy is probably better (by abstracting the way it replaces the response so it can be reused here), since I think we want them to be equivalent. But maybe I'm missing something?